eo-maven-plugin/src/main/java/org/eolang/maven/UnphiMojo.java

```diff
@@ -62,6 +62,12 @@
     threadSafe = true
 )
 public final class UnphiMojo extends SafeMojo {
+
+    /**
+     * Subdirectory for parsed cache.
+     */
+    static final String CACHE = "unphied";
```

Comment:
@ArtemGet don't keep it here, since you don't use it. Put it into the test for now.